trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table.njk

```diff
@@ -1,57 +1,99 @@
-            </a>
-          {% endif %}
-        </td>
-      </tr>
-    {% endfor %}
-    </tbody>
-  </table>
+      <tbody class="govuk-table__body">
+      {% for summaryItem in reportReconciliationSummary %}
+        {% set checkboxId = ['set-status--bankId-', summaryItem.bank.id] | join %}
+        {% if summaryItem.reportId %}
+          {% set checkboxId = ['set-status--reportId-', summaryItem.reportId] | join %}
+        {% endif %}
+
+        <tr class="govuk-table__row" data-cy="utilisation-report-reconciliation-table-row-bank-{{ summaryItem.bank.id }}">
+          <th
+            scope="row"
+            class="govuk-table__header govuk-!-font-weight-regular"
+            data-sort-value="{{ summaryItem.bank.name }}"
+          >
+            <div class="govuk-checkboxes govuk-checkboxes--small">
+              <div class="govuk-checkboxes__item" data-cy="checkbox--{{ id }}">
+                <input class="govuk-checkboxes__input govuk-!-top-0 bank-select-checkbox-input" id="{{ checkboxId }}" name="{{ checkboxId }}" type="checkbox"/>
+                <label class="govuk-label govuk-checkboxes__label govuk-!-padding-0 bank-select-checkbox-label" for="{{ checkboxId }}">
+                  <a class="govuk-link govuk-link--no-visited-state gov-table_cell-vertical-align-middle" href="#">{{ summaryItem.bank.name }}</a>
+                </label>
+              </div>
+            </div>
+          </th>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.displayStatus }}">
+            {{ reconciliationStatus.render({
+              status: summaryItem.status,
+              displayStatus: summaryItem.displayStatus
+            }) }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.dateUploaded }}">
+            {{ summaryItem.formattedDateUploaded }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.totalFeesReported }}">
+            {{ summaryItem.totalFeesReported }}
+          </td>
+          <td class="govuk-table__cell" data-sort-value="{{ summaryItem.reportedFeesLeftToReconcile }}">
+            {{ summaryItem.reportedFeesLeftToReconcile }}
+          <td class="govuk-table__cell">
+            {% if summaryItem.downloadPath %}
+              <a
+                class="govuk-link"
+                href="{{ summaryItem.downloadPath }}"
+                aria-label="Download {{ summaryItem.bank.name }} report"
+              >
+                Download
+              </a>
+            {% endif %}
+          </td>
+        </tr>
+      {% endfor %}
+      </tbody>
+    </table>
+
+    <p class="govuk-body">
+      When a report has been reconciled, select it and mark it as completed. If it needs to be returned to its previous state, mark it as not completed.
+    </p>
+
+    <div class="govuk-button-group">
+      {{ govukButton({
+        text: "Mark report as completed",
+        classes: "govuk-!-margin-right-1",
```

Comment:
Looks like this margin needs to be removed again https://github.com/UK-Export-Finance/dtfs2/pull/2489#discussion_r1439575137